cli/cli/kurtosis_config/resolved_config/kurtosis_config.go

```diff
@@ -20,8 +20,9 @@ const (
+	defaultShouldTurnOffDefaultLogsSink = false
```

Comment:
1. Same comment about negation
2. Same question about if this value should be used when doing the config parsing (maybe not; I'm not sure)